python/python/glide/async_commands/core.py

```diff
@@ -1002,6 +1002,36 @@ async def lpop_count(self, key: str, count: int) -> Optional[List[str]]:
             await self._execute_command(RequestType.LPop, [key, str(count)]),
         )
 
+    async def blpop(self, keys: List[str], timeout: float) -> Optional[List[str]]:
+        """
+        Pops an element from the head of the first list that is non-empty, with the given keys being checked in the
+        order that they are given.
+        Blocks the connection when there are no elements to pop from any of the given lists.
+
+        See https://valkey.io/commands/blpop for details.
+
+        Notes:
+            1: BLPOP is a client blocking command, see https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands for more details and best practices.
+            2: When in cluster mode, all keys must map to the same hash slot.
```

Comment:
```suggestion
            2: When in cluster mode, all `keys` must map to the same hash slot.
```

Reply:
I've changed these docs to align with existing doc patterns. The patterns I've noticed and followed are:
- use angle quotes only for argument names, unless the argument is plural (eg `keys`, `members` etc) or we are describing the arg itself in the "Args" section
- do not use "Note" or "Notes" as this isn't used elsewhere, instead include the hash slot and/or blocking command warnings on their own separated lines in the doc